provision/kubernetes/minikube/README.markdown

```diff
@@ -0,0 +1,830 @@
+# Introduction
+
+The goal of this document is to provide guidance in the setup,
+administration, and operation of various components of the HashiStack
+when running on or integrating with the
+[Kubernetes](https://kubernetes.io) cluster management framework.
+
+The document assumes Enterprise offerings of the various HashiStack
```

Comment:
Is there anything in here that requires Enterprise offerings? It doesn't look like any Enterprise specific features for Vault or Consul are being used. If not, I would just mention the path for them to optionally build Enterprise images (which you created) - the wording makes it sound like it's a prerequisite or it won't work.

Reply:
Thinking back on it now...I think its probably just the Auto Unseal stuff. So it could rely on OSS and then just call out to Enterprise for the HSM disussion.

Reply:
Yeah I like that.